Documentation site (#327)
koskimas/kysely: site/docs/installation.md

---
sidebar_position: 2
title: 'Getting started'
---

# Getting started

> Kysely currently works on PostgreSQL, MySQL and SQLite. You can install by running:

Reviewer: ```suggestion
Kysely currently works out of the box on PostgreSQL, MySQL and SQLite. You can install by running:
```
wdyt?

Reviewer 2: Alternatively..

```suggestion
Kysely has built-in support for PostgreSQL, MySQL and SQLite. You can install by running:
```
